xbmc/cores/VideoPlayer/VideoRenderers/CMakeLists.txt

```diff
@@ -39,7 +39,12 @@ if(OPENGL_FOUND)
                       FrameBufferObject.h)
 endif()
 
-if(OPENGLES_FOUND)
+if(CORE_PLATFORM_NAME STREQUAL android OR
+   CORE_PLATFORM_NAME STREQUAL ios OR
+   CORE_PLATFORM_NAME STREQUAL gbm OR
+   CORE_PLATFORM_NAME STREQUAL imx OR
+   CORE_PLATFORM_NAME STREQUAL mir OR
+   CORE_PLATFORM_NAME STREQUAL wayland)
```

Comment:
wayland can use OpenGL or GLES

Reply:
I really don't know what you want. In this case I should leave the OPENGLES_FOUND statement,

Reply:
I just point out (once again) that our build system does not fit our requirements. The current issues you encounter are a clear indicator for this. 

Ideally a platform maintainer should be able to include platform dependent code in the build. The platform maintainer is the only one how really knows that.